src/Actions/Order/CreateOrder.php

```diff
@@ -27,6 +27,12 @@ public function execute(): Order
         $this->data['currency_id'] = $this->data['currency_id'] ?? Currency::query()->first()?->id;
         $addresses = Arr::pull($this->data, 'addresses', []);
 
+        if (! ($this->data['address_delivery']['id'] ?? false) && ($this->data['address_delivery_id'] ?? false)) {
+            $this->data['address_delivery_id'] = null;
+        } else {
```

Comment:
if address_deliver_id = null and address_delivery is not set, the else condition is met, and therefore and exception will be thrown in the else block.